include/pfs/drp/stella/SpectraTemplates.hpp

```diff
@@ -397,7 +405,14 @@ bool pfs::drp::stella::Spectrum<SpectrumT, MaskT, VarianceT, WavelengthT>::ident
     _dispCorControl = tempDispCorControl.getPointer();
 
     ///for each line in line list, find maximum in spectrum and fit Gaussian
-    ndarray::Array< double, 1, 1 > D_A1_GaussPos = hIdentify( lineList );
+    ndarray::Array< double, 1, 1 > D_A1_GaussPos;
+    try{
+        D_A1_GaussPos = hIdentify( lineList );
+    } catch (...){
+        cout << "hIdentify threw an error";
```

Comment:
Please do not write to `cout/cerr`. 

Why aren't you including the error from hIdentify in the log message (augmented by anything you know that hIdentify doesn't)?